shaal/DrupalPod
# Fix drush cr

# The Problem/Issue/Bug
Running `drush cr` on core development setup, with contrib modules, make the contrib modules disappear.

## How this PR Solves The Problem
Implement the fix from @joachim-n's `drupal-core-development-project`
https://github.com/joachim-n/drupal-core-development-project/commit/e3f5c7f362039dce851a4b8da53abe443d423d35

## Manual Testing Instructions
Open -
https://gitpod.io/#DP_PROJECT_NAME=drupal,,DP_PROJECT_TYPE=project_core,DP_MODULE_VERSION=10.0.x,DP_CORE_VERSION=10.0.x,DP_INSTALL_PROFILE=standard/https://github.com/shaal/drupalpod/pull/90

Confirm admin_toolbar is installed and enabled.
Run `drush cr` in terminal.
Confirm admin_toolbar is still installed and enabled.

## Related Issue Link(s)

## Release/Deployment notes



<a href="https://gitpod.io/#https://github.com/shaal/DrupalPod/pull/90"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
